torchaudio/sox_effects.py

```diff
@@ -77,9 +78,14 @@ class SoxEffectsChain(object):
 
     """
 
-    EFFECTS_UNIMPLEMENTED = set(["spectrogram", "splice", "noiseprof", "fir"])
+    EFFECTS_UNIMPLEMENTED = {"spectrogram", "splice", "noiseprof", "fir"}
```

Comment:
Is there a reason you are changing this?

Reply:
flake8 -> `torchaudio/sox_effects.py:81:29: C405 Unnecessary list literal - rewrite as a set literal.`

Reply:
Fair enough :)